zapcli/cli.py

```diff
@@ -430,6 +430,54 @@ def report(zap_helper, output, output_format):
     console.info('Report saved to "{0}"'.format(output))
 
 
+@cli.command('test-alerts')
+@click.option('--config-file', '-c', default=None, type=str,
+              help='config file to use to INFO, IGNORE or FAIL warnings')
+@click.option('--config-url', '-u', default=None, type=str,
+              help='URL of config file to use to INFO, IGNORE or FAIL warnings')
+@click.option('--generate', '-g', is_flag=True, default=False,
+              help='generate default config file (all rules set to WARN)')
+@click.option('--info', '-i', is_flag=True, default=False,
+              help='default rules not in the config file to INFO')
+@click.option('--level', '-l', default='PASS', type=str,
+              help='minimum level to show: PASS, IGNORE, INFO, WARN or FAIL,' +
+              'use with -s to hide example URLs. Default: PASS')
+@click.option('--quiet', '-q', is_flag=True, default=False, type=str,
+              help='short output format - dont show PASSes or example URLs')
+@click.pass_obj
+def test_alerts(zap_helper, **options):
+    """Test zap alert and print the results, according to the configuration"""
```

Comment:
Maybe: Fetch all the alerts from zap, and set the exit code according to the severity of the findings. The severity of the alerts is set via configuration (using --generate to generate the config file). The exit code indicate the level: 0 - passed, 1 - failure, 2 - warning, 3 - ignore (meaning, all alerts defined found as ignored).